spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -130,6 +130,25 @@
     end
 
     context 'user is not signed in' do
+      context 'without valid acr_values' do
+        before { params.delete(:acr_values) }
+
+        it 'handles the error and does not blow up' do
+          action
+
+          expect(response).to redirect_to(/^#{params[:redirect_uri]}/)
```

Comment:
What is the reason for redirecting in this scenario? Shouldn't all invalid requests be treated the same? Also, what is the user experience here (in general, not this particular test)? If bad requests are (usually) the fault of the SP, does it make sense to display an error page to the user for something they did not cause? Does the error page have a helpful message explaining what happened and what they can do?